<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
<<<<<<< Updated upstream:ui/clinicas_information.ui
    <height>333</height>
=======
    <height>415</height>
>>>>>>> Stashed changes:ui/animal_information.ui
   </rect>
  </property>
  <property name="windowTitle">
   <string>Información de Clínica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="volverButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modificarButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrarButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="leftDisplay">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="name">
           <property name="text">
            <string>Nombre de la Clínica</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="id">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
<<<<<<< Updated upstream:ui/clinicas_information.ui
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
=======
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="species">
           <property name="text">
            <string>Especie</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="owner">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
>>>>>>> Stashed changes:ui/animal_information.ui
         </property>
         <property name="text">
          <string>Dueño</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightDisplay">
       <item>
<<<<<<< Updated upstream:ui/clinicas_information.ui
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="municipio">
           <property name="text">
            <string>Municipio</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="provincia">
           <property name="text">
            <string>Provincia</string>
           </property>
          </widget>
         </item>
        </layout>
=======
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Descripción:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="description">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
        </widget>
>>>>>>> Stashed changes:ui/animal_information.ui
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
